packages/client/components/Checkbox.tsx

```diff
@@ -1,29 +1,40 @@
-import React from 'react'
 import styled from '@emotion/styled'
+import {CheckBox, CheckBoxOutlineBlank, IndeterminateCheckBox} from '@mui/icons-material'
+import React from 'react'
 import {PALETTE} from '../styles/paletteV3'
-import Icon from './Icon'
-
 interface Props {
   active: boolean | null
   className?: string
   disabled?: boolean
   onClick?: (e: React.MouseEvent) => void
 }
 
-const StyledIcon = styled(Icon)<{disabled: boolean | undefined}>(({disabled}) => ({
+const StyledIcon = styled('div')<{disabled: boolean | undefined}>(({disabled}) => ({
   color: PALETTE.SLATE_600,
   cursor: disabled ? 'not-allowed' : 'pointer',
+  height: 24,
+  width: 24,
   display: 'block',
   opacity: disabled ? 0.38 : 1,
   userSelect: 'none'
 }))
 
 const Checkbox = (props: Props) => {
   const {active, className, disabled, onClick} = props
-  const icon = active ? 'check_box' : active === false ? 'check_box_outline_blank' : 'indeterminate_check_box'
+  const icon = active
+    ? 'check_box'
```

Comment:
-1 You could assign the icon component directly here instead of doing the extra step with a map:
```ts
const icon = active ? <CheckBox /> : active === false ? <CheckBoxOutlineBlank /> : <IndeterminateCheckBox />
```
and later just use `{icon}` or move the branch directly to line 31